src/install.ts

```diff
@@ -17,9 +17,10 @@ export async function runCommand(command: string): Promise<string> {
 }
 
 async function installOnLinux(version: string): Promise<string> {
-  const system = runCommand('uname -s')
-  const hardware = runCommand('uname -m')
-  const url = `https://github.com/docker/compose/releases/download/${version}/docker-compose-${await system}-${await hardware}`
+  const system = await runCommand('uname -s')
+  const hardware = await runCommand('uname -m')
+  const casedSystem = version.startsWith('2') ? `${system[0].toUpperCase()}${system.slice(1)}` : system
```

Comment:
`version.startsWith('2')` will not work for v3 or later major versions.
Could you consider making it more strict, just like `parseInt(version.split('.')[0], 10) >= 2`?

Reply:
We still don't know release naming convention for v3 onward however.

Reply:
But you're right that it's highly likely that more recent format will stay.